anyway/models.py

```diff
@@ -167,14 +182,23 @@ def get_id(self):
         return self.id
 
 
-class Role(Base, RoleMixin):
-    __tablename__ = "roles"
-    id = Column(Integer(), primary_key=True)
-    name = Column(String(80), unique=True)
+class Roles2(Base):
+    __tablename__ = "roles2"
+    id = Column(Integer(), primary_key=True, autoincrement=True, nullable=False)
+    name = Column(String(127), unique=True, index=True, nullable=False)
```

Comment:
Do we need the name in the DB? we do have it as a constant in the code. If we do need it in the DB, placing it in a different translation table, (just id and name), will make queries of this table faster.

Reply:
We do need the names of the roles in the DB as in the future we might add an API of `create_role`, so admins can create preference and setting dynamically.
Some of the roles are hardcoded (like the `admins` role) and so there is a code specifically written for this role, this is why i created a constant class of roles in the code.

I don't believe that there will be a lot of roles in the DB - less than 50, and when thinking about it i don't use the `id` field at all - so I can remove the id field and make the `name` field as primary key.

What do you think? - remove `id` and replaced it with `name`? or create a different translation table with `name` and `id`?

Reply:
I prefer the id and translation table, mainly due to the big runing time difference of text in the database. It is a small investment at the beginning, but almost no additional cost going on. We can write a function to get the name of a given id, so it will be quicker for new developers to maintain it.

Reply:
I agree with @BarVolunteering that we won't have a lot of roles, and also agree with @ziv17  that creating a translation table is a better modeling. I leave it up to you @BarVolunteering to decide on this one.

Reply:
In all cases where i use the `id` i also use `name` - so in all cases there will be a join between the translation table and the data table and the performance impact will be negligible.